src/Microsoft.Security.Utilities.Core/Releases.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Security.Utilities;
+
+/// <summary>
+/// This class contains the version numbers of the library releases. Prior to
```

Comment:
nit: everything after the first sentence might be more at home in `<remarks>` rather than `<summary>`.